src/platform/qemu/run.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env nix-shell
+#!nix-shell --pure -p qemu -i bash
+
```

Comment:
Why do you have this file duplicated between `framework/platform/qemu` and `src/platform/qemu` ?

Reply:
Nice catch, removed the `src/platform` directory. Thanks!